code/game/objects/items/devices/portable_recharger.dm

```diff
@@ -0,0 +1,330 @@
+/obj/item/portable_recharger
+	name = "backpack recharger"
+	icon = 'icons/obj/rechargers.dmi'
+	icon_state = "backcharger"
+	base_icon_state = "backcharger"
+	desc = "A portable backpack charging dock for energy based weaponry, PDAs, and other devices."
+	inhand_icon_state = "backcharger"
+	lefthand_file = 'icons/mob/inhands/equipment/security_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
+	slot_flags = ITEM_SLOT_BACK
+	force = 5
+	throwforce = 6
+	w_class = WEIGHT_CLASS_BULKY
+	///Our power cell
+	var/obj/item/stock_parts/cell/high/cell
```

Comment:
```suggestion
	var/obj/item/stock_parts/cell/cell
```
No point in saying it MUST be a high level cell, because...

Reply:
if you want it to coantain a high cell, simply initialize it like

`var/obj/item/stock_parts/cell/cell = var/obj/item/stock_parts/cell/high/cell`